Makefile

```diff
@@ -1,5 +1,6 @@
 
-DOCKER ?= $(shell which docker)
+DOCKER ?= $(shell command -v docker)
```

Comment:
@wking It looks like it should work fine, but I got error `make: command: Command not found`, I have to change it back to `which`, any insight on this? Maybe related to bash version?